server/services/v1/database/sessions.go

```diff
@@ -33,6 +38,11 @@ import (
 	ulog "github.com/tigrisdata/tigris/util/log"
 )
 
+const (
+	NamespaceConsultation = "namespace_consultation"
```

Comment:
Would "federation" be a better term here?
WDYT @JigarJoshi @himank @ovaistariq

Reply:
Here we are checking from a central place so probably not federation.

Reply:
`NamespaceConsultation` or `MetadataClusterConsultation` sounds good to me.

Reply:
ok.

Reply:
Discussed offline and decided to rename it to `NamespaceLocalization`